connection.go

```diff
@@ -124,6 +125,12 @@ func (e errConnectionUnknownState) Error() string {
 	return fmt.Sprintf("connection is in unknown state: %v at %v", e.state, e.site)
 }
 
+// SendBufferSizeOverride is used for overriding per process name prefix send buffer channel size.
```

Comment:
nit: comments can be improved, and call out this is per-connection, maybe: 
`SendBufferSizeOverride is used for overriding per-process send buffer channel size for a connection, using process name prefix matching.`